package-lock.json

```diff
@@ -388,6 +388,15 @@
         "@babel/helper-plugin-utils": "^7.0.0"
       }
     },
+    "@babel/plugin-syntax-dynamic-import": {
```

Comment:
Is this currently used?

Reply:
Hmmm. I must have manually removed from the package.json file and thus the lock wasn't cleaned up. I will fix and rebase.

Reply:
Reolved